README.md

```diff
@@ -375,6 +376,35 @@ To create the secret, you can use the command below:
 ```sh
 # kubectl create secret generic <resourcename>-ldap-ca-cert --from-file=ldap-ca.crt=<PATH/TO/YOUR/CA/PEM/FILE>
 ```
+### Tower Custom Settings
+
+If the variable `tower_custom_settings` is provided, the operator will look for the ConfigMap specified in the field for a custom.py key.
+
+| Name                             | Description                             | Default |   Key    |
+| -------------------------------- | --------------------------------------- | --------| ---------|
+| tower_custom_settings            | ConfigMap which stores custom settings  |  ''     | custom.py|
+
+To create a ConfigMap, you can use the example yaml file as a reference:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: awx-custom-settings
+  namespace: default
+data:
+  custom.py: |
```

Comment:
I'm wondering if we can make this a bit more flexible by making this key the full path the file should be bind-mounted to. So in your case here it would be:

```suggestion
  `/var/lib/awx/custom.py`: |
```

If we did this I think it would fully satisfy the ask in https://github.com/ansible/awx-operator/issues/175

Reply:
Stating the obvious: If we did this we would need to rework the logic around how this is consumed inside of the Operator. LMK if you have any questions about what I mean.